src/Dfc.CourseDirectory.Core/DataStore/Sql/QueryHandlers/GetProviderMissingPrimaryContactReportHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using Dapper;
+using Dfc.CourseDirectory.Core.DataStore.Sql.Queries;
+using Dfc.CourseDirectory.Core.Models;
+
+namespace Dfc.CourseDirectory.Core.DataStore.Sql.QueryHandlers
+{
+    public class GetProviderMissingPrimaryContactReportHandler : ISqlAsyncEnumerableQueryHandler<GetProviderMissingPrimaryContactReport, ProviderMissingPrimaryContactReportItem>
+    {
+        public async IAsyncEnumerable<ProviderMissingPrimaryContactReportItem> Execute(SqlTransaction transaction, GetProviderMissingPrimaryContactReport query)
+        {
+            var sql = @$"
+SELECT      p.Ukprn as ProviderUkprn,
+            p.ProviderName,
+            p.ProviderType,
+            p.ProviderStatus,
+		    cu.LiveCourseCount,
```

Comment:
Do we need these 3 columns?

Reply:
I've removed them now - was debug columns :)